feature: Add uwu command

MadHyena opened the pull request:
The >uwu <msg> command will post the UwU speak version of msg in the current channel.

MadHyena committed:
feature: Add uwu command

The >uwu <msg> command will post the UwU speak version of msg in
the current channel, if it is the general channel.

zuzuvelas commented on src/commands/uwu.ts:
```diff
@@ -0,0 +1,46 @@
+class UwU {
+
+  private general_channel = '103678524375699456';
+  private channel: TextChannel | TextBasedChannel | null = null;
```
It looks like this is unused so you can just remove it... unless I'm blind which is also possible

zuzuvelas commented on src/commands/uwu.ts:
```diff
@@ -0,0 +1,46 @@
+      return command.sendUsageSyntax()
+    }
+
+    if (command.message.channel.id !== this.general_channel) {
```
This command should work outside of general. I think the original limitations were if it was IN general then it had to be less than 200 characters otherwise it worked fine.

MadHyena committed:
refactor: Make the uwu command usable outside of general

Remove the general channel check.
Also removed the unused general channel id member.
